docs/index.rst

```diff
@@ -5,65 +5,84 @@ Tahoe-LAFS
 .. Please view a nicely formatted version of this documentation at
    http://tahoe-lafs.readthedocs.io/en/latest/
 
-Contents:
+Tahoe-LAFS is a Free and Open decentralized cloud storage system.
+It distributes your data across multiple servers.
+Even if some of the servers fail or are taken over by an attacker,
+the entire file store continues to function correctly,
+preserving your privacy and security.
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
+   :caption: Preliminaries
 
    about
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Running Tahoe-LAFS
+
    INSTALL
    running
-   magic-wormhole-invites
    configuration
-   architecture
-
+   servers
    frontends/CLI
-   frontends/webapi
    frontends/FTP-and-SFTP
    frontends/download-status
-
+   magic-wormhole-invites
+   anonymity-configuration
    known_issues
-   contributing
-   CODE_OF_CONDUCT
-   release-checklist
 
-   servers
+.. toctree::
+   :maxdepth: 1
+   :caption: Technical Notes
```

Comment:
Technical notes for whom?  Developers?  End-users?  I think the docs here now are a mix.  For example, if I want to use Tahoe-LAFS maybe I want to configure and run a helper or customize my convergence secret.

The "Running[/Using] Tahoe-LAFS" section above strikes me as a good kind of organization for the documentation.  If I am new to Tahoe-LAFS and approaching the docs, at some point I will decide I want to use it and that section title guides me to exactly the part of the docs I need to look at.

How can we do something similar for this section title (and which docs in it should we move elsewhere to accommodate that)?

Reply:
Yes, it is a weird mix.  I believe documents under this section would be of interest to both developers and some end-users (curious minds, people who want to integrate Tahoe-LAFS with some other system, system administrators who needs to know Tahoe-LAFS better, potential contributors, etc.)

Can we call this section "Tahoe-LAFS in depth"?

Some of the things here ("Avoiding Write Collisions", "Things to be Careful..." etc) could be moved elsewhere, and/or combined, but that probably should be separate PR.

Reply:
I would rename this to "About Tahoe-LAFS" and if there are many parallel articles with varying complexity, then maybe "About Tahoe-LAFS in depth".

Reply:
Renamed it to "Tahoe-LAFS in Depth".